.github/workflows/lint.yml

```diff
@@ -43,25 +44,26 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-          linter: [cppcheck, cpplint, uncrustify]
+        linter: [cppcheck, cpplint, uncrustify]
+
     steps:
     - uses: actions/checkout@v2
     - uses: ros-tooling/action-ros-lint@v0.1
       with:
         linter: ${{ matrix.linter }}
         distribution: rolling
         package-name: |
-            rosbag2_compression
-            rosbag2_compression_zstd
-            rosbag2_cpp
-            rosbag2_examples_cpp
-            rosbag2_py
-            rosbag2_storage
-            rosbag2_storage_sqlite3
-            rosbag2_test_common
-            rosbag2_tests
-            rosbag2_transport
-            shared_queues_vendor
+          rosbag2_compression
+          rosbag2_compression_zstd
+          rosbag2_cpp
+          rosbag2_examples_cpp
+          rosbag2_py
+          rosbag2_storage
+          rosbag2_storage_sqlite3
+          rosbag2_storage_mcap
+          rosbag2_test_common
+          rosbag2_tests
+          rosbag2_transport
```

Comment:
Curious about why `shared_queues_vendor` was removed from the list?

Reply:
Good catch, that was an oversight. I originally removed `shared_queues_vendor` when I split up the packages into a matrix, because then each package got its own lint job and `shared_queues_vendor` failed because there's no code in that package to lint. I can re-add it now for completeness, even though it does nothing in this context.